docs/usage/mnist.rst

```diff
@@ -50,28 +51,38 @@ Assess the model's accuracy.
 
     0.8269000053405762
 
-Create an ``image.csv`` file which contains the names of the images (with their extension .mnistpng) to predict on.
-
-.. note::
-    Make sure to download each image and save them with the `.mnistpng` extension.
+Create an ``image.csv`` file which contains the names of the images (with their extension .png) to predict on.
 
 .. literalinclude:: /../examples/MNIST/image_file.sh
 
+In the config file, using the ``dataflow create`` command we create a DataFlow
+consisting of 3 operations: resize, multiply and associate_definition. We
+edit the DataFlow to have the data input of the resize operation come
+from a list in seed containing ``image`` feature, have the multiplicand input of
+multiply operation come from the output (resized_array) of the resize operation
+and to have the input of the associate_definition operation come from the output
+(product) of the multiply operation.
+
+.. literalinclude:: /../examples/MNIST/create_dataflow.sh
```

Comment:
We should really make this a separate file. Due to what yash talked about here: https://gitter.im/dffml/community?at=5ee5fd4f7b6da9126a942995

Reply:
We still need to do this